src/fastfs.js

```diff
@@ -353,7 +339,7 @@ function makeReadCallback(fd, predicate, callback) {
 }
 
 function isDescendant(root, child) {
-  return child.startsWith(root);
+  return path.resolve(child).startsWith(root);
```

Comment:
can we avoid doing this? path.resolve is pretty slow and we call this 50k times on our (tiny!) react-native internal codebase.